src/Raven.Server/Config/Categories/PerformanceHintsConfiguration.cs

```diff
@@ -2,11 +2,24 @@
 using Raven.Server.Config.Attributes;
 using Raven.Server.Config.Settings;
 using Sparrow;
+using Sparrow.LowMemory;
+using Sparrow.Platform;
 
 namespace Raven.Server.Config.Categories
 {
     public class PerformanceHintsConfiguration : ConfigurationCategory
     {
+        public PerformanceHintsConfiguration()
+        {
+            if (PlatformDetails.RunningOnLinux)
+            {
+                var threshold = new Size(8, SizeUnit.Gigabytes);
+                MinSwapSize = MemoryInformation.TotalPhysicalMemory < threshold 
```

Comment:
Are we sure that we want to raise an alert when on 128GB machine swap is lower than 64GB?

Reply:
No, if the half of the physical memory is greater than 8GB so we set it to 8GB

Reply:
You are correct 🥇